azkaban-common/src/main/java/azkaban/jobExecutor/AbstractProcessJob.java

```diff
@@ -184,35 +186,38 @@ public Props loadOutputFileProps(final File outputPropertiesFile) {
       }
       return outputProps;
     } catch (final FileNotFoundException e) {
-      this.log.info(String.format("File[%s] wasn't found, returning empty props.",
-          outputPropertiesFile));
+      this.info(String.format("File[%s] wasn't found, returning empty props.", outputPropertiesFile));
       return new Props();
     } catch (final Exception e) {
-      this.log.error(
-          "Exception thrown when trying to load output file props.  Returning empty Props instead of failing.  Is this really the best thing to do?",
+      this.error(
+          "Exception thrown when trying to load output file props.  Returning empty Props instead of failing. Is this really the best thing to do?",
           e);
       return new Props();
     } finally {
       IOUtils.closeQuietly(reader);
     }
   }
 
+  /**
+   * TODO: Review to see if this function can be private
```

Comment:
i don't see it's being used outside of the class, let's make it private?

Reply:
sure. I didn't make it private in this RB just in case there are some dependencies which I don't know.

Reply:
converted to private